### DISCUSSION TITLE
NCP Spinel support for external commissioning

### DISCUSSION THREAD

**Discussion by victormsc** [openthread/openthread#8302] [Q&A] [answered]
Hello,

please, could you tell me if the NCP Spinel protocol supports the use of an external commissioner.

If the answer is yes, could you tell me the Spinel properties involved or any guide where this topic is discussed?

Thank you very much.

**Comment by jwhui** [comment, marked as answer]
The NCP model does support an external commissioner. In this case, the NCP is responsible for handling the Border Agent DTLS session with the external commissioner. To support this, the NCP uses the `UDP_FORWARD` feature, where UDP sent to the MeshCoP UDP port are forwarded to the NCP over spinel.
